index.js

```diff
@@ -194,17 +194,30 @@ class CoverageSonar extends CoverageBase {
      * @param   {String}  config.jobId      Screwdriver job ID
      * @param   {String}  config.startTime  Job start time
      * @param   {String}  config.endTime    Job end time
-     * @return  {Promise}                   An object with coverage badge link and project link
+     * @return  {Promise}                   An object with:
+     *                                        - coverage percentage
+     *                                        - project url
+     *                                        - Screwdriver API coverage auth url
+     *                                        - coverage host
      */
     _getInfo({ jobId, startTime, endTime }) {
         const componentId = encodeURIComponent(`job:${jobId}`);
-        const projectUrl = `${this.config.sonarHost}/dashboard?id=${componentId}`;
+        const projectUrl = `${this.config.coverageHost}/dashboard?id=${componentId}`;
+
+        if (jobId && startTime && endTime) {
```

Comment:
is this end time for job ? This means that without all bookends being completed we won't show coverage numbers. We should show the number as soon as coverage bookend is done.  This should be made dependent on coverage step's start and end time.

Reply:
This method was already based on the first and last steps' start and end time, respectively. The functionality isn't changing, I'm just overloading the method to return basic Sonar coverage data when the steps are not finished running yet so the launcher can make this call and get that data.